client/flutter/assets/onboarding/iso_countries.en.yaml

```diff
@@ -0,0 +1,499 @@
+countries:
+  - name: Afghanistan
```

Comment:
easiest to sort by the key (the code) instead of th localizabl name

Reply:
I didn't do any sorting on this at all, I just kept it in exactly the order it was listed on the ISO site. Would you like me to sort?